deploy-agent/deployd/agent.py

```diff
@@ -87,10 +88,22 @@ def load_status_file(self):
         self._config.update_variables(self._curr_report)
 
     def first_run(self):
```

Comment:
you can make this an `@property` for clarity for the user, although the practical value is dubious without a setter

Reply:
this change is definitely recommended for working with `_first_run` private state, then you can drop the `()` off all references to `first_run` in the codebase. I think this is a good time to do it since you are testing all the cases anyways.

Reply:
Both great suggestions.  Added.  Thank you!